contracts/stake/src/multitest/distribution.rs

```diff
@@ -145,200 +119,14 @@ fn multiple_distribution_flows() {
         .create_distribution_flow(
             "admin",
             &members[0],
-            AssetInfo::Token(dex_token.to_string()),
```

Comment:
Why did you remove this whole part of the test?

Reply:
but right now we also should have the same - a total of 3 distributions for juno, luna and dex tokens

Reply:
brought back a bunch of tests. 

tests not brought back, because their msgs are no longer used:
- delegate.rs: delegate_as_properly_assigned()
- distribution.rs: funds_withdrawal_delegation(), rebond_works(), rebond_multiple_works()
- quick_unbond.rs: quick_unbond_case() renamed to unbond() removed now; non_staker_works(), unbonder_permission_check()
- unbond_all.rs: mass_delegation_with_unbond_all_flag(), rebond_with_unbond_all_flag()